swig/lua52/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+add_definitions(-DSWIG)
+add_definitions(-DSWIGLUA)
+
+project(Sysrepo_Lua)
+
+add_library(Sysrepo_Lua SHARED ${SYSREPO_CPP_SOURCES})
+
+# include custom Modules
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/CMakeModules/")
+
+include_directories(${CMAKE_SOURCE_DIR})
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
+target_link_libraries(Sysrepo_Lua sysrepo)
+
+if(NOT BIN_INSTALL_DIR)
+	set(BIN_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/bin)
+endif()
+
+set(LUA_SWIG_BINDING_52 libsysrepoLua52)
+
+if (SWIG_FOUND)
+    include(${SWIG_USE_FILE})
+    find_package(Lua 5.2 REQUIRED)
```

Comment:
Hi Rastislav,
can you take a look at the new commit with the proposed changes.
Regards,
Mislav